packages/editor/src/types/config.ts

```diff
@@ -40,11 +41,24 @@ export type Subsection =
   | 'Layout'
   | 'Icon';
 
+export type Section = {
+  name: string;
+  icon: IvyIcons;
+};
+
+export const sections: Section[] = [
+  { name: 'Properties', icon: IvyIcons.List },
+  { name: 'Layout', icon: IvyIcons.DialogLayout },
+  { name: 'Confirm', icon: IvyIcons.Comment }
+];
+
+type SectionId = (typeof sections)[number]['name'];
```

Comment:
now sectionid is just a string, you missed this section:
```suggestion
export const sections = [
  { name: 'Properties', icon: IvyIcons.List },
  { name: 'Layout', icon: IvyIcons.DialogLayout },
  { name: 'Confirm', icon: IvyIcons.Comment }
]  as const satisfies Array<Section>;

type SectionId = (typeof sections)[number]['name'];
```